sources/Application/Views/ModalDialogs/PagedImportSampleDialog.cpp

```diff
@@ -4,14 +4,11 @@
 #include "pico/multicore.h"
 #include <memory>
 
-#define LIST_SIZE 15
+#define LIST_SIZE PAGED_PAGE_SIZE
```

Comment:
Why not use PAGED_PAGE_SIZE directly where needed in this file?

Reply:
@democloid  I wanted to keep the distinction just to make it a bit clearer in the code that the 2 values are equal at the moment but have different meanings between here and the file system code, but I can remove LIST_SIZE if you would prefer?

Reply:
I guess I don't see when these two values would differ, given that we chose paged_page_size precisely to match our screen (or list) size. But don't feel too strongly about it.

Reply:
No its a good point, thinking about it some more, I think this makes things more complicated then needed, so I've switched to just using the single `#define`